curriculum/challenges/english/25-front-end-development/quiz-dom-manipulation-and-click-event-with-javascript/66edd07682767adff3a6231e.md

```diff
@@ -364,29 +331,29 @@ Which event is triggered when all the HTML page's elements have been parsed by t
 
 #### --text--
 
-Which event is triggered when the HTML page and it's resources are fully loaded?
+Which of the following is a read-only property that represents the inline style of an element?
```

Comment:
```suggestion
Which of the following is a property that represents the inline style of an element?
```

It's halfway read-only. You can't assign a style object directly, but you *can* assign a string to the property to completely overwrite existing inline styles.